spec/security_alert_handler_spec.rb

```diff
@@ -0,0 +1,130 @@
+require "spec_helper"
```

Comment:
Thanks for adding these tests. Please could you merge them into the same commit as the `SecurityAlertHandler` class being added? They should be reviewed together rather than in isolation.

To prevent the commit from getting too big, I'd split out the remaining changes from the `Add security alerts handling for Dependabot PRs` commit, i.e. just have a `Create SecurityAlertHandler` commit (adding the class and tests, with a commit message along the lines of "This class will be used in the next commit") and then a separate `Add security alerts handling for Dependabot PRs` which uses the SecurityAlertHandler.

Reply:
Nice tests 🎉